src/main/java/HttpGetRequest.java

```diff
@@ -0,0 +1,40 @@
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+class HttpGetRequest extends UserInterface{
+    StringBuffer response = new StringBuffer();
+
+    void httpGetRequest() throws Exception {
+        LocalDate localDate = LocalDate.now();
+        LocalDate yesterday = localDate.minusDays(2);
```

Comment:
Why do you call you variable yesterday if you use the day before yesterday date. 
I suggest renaming this variable to  "beginDayDate" or else. And please don't set variables directly, use constant's for this.